backend/plugin/db/redis/sync.go

```diff
@@ -17,38 +18,17 @@ import (
 // SyncInstance syncs the instance metadata.
 func (d *Driver) SyncInstance(ctx context.Context) (*db.InstanceMetadata, error) {
 	var instance db.InstanceMetadata
-	var databaseCount int
-
-	version, err := d.getVersion(ctx)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to get server version")
-	}
-	instance.Version = version
```

Comment:
The version is displayed on the frontend. Please undelete these lines.

Reply:
ok ,i have add these lines.